pkgbot/__init__.py

```diff
@@ -317,7 +317,7 @@ def process_request(data):
         for distro in createrepo_distros:
             rpm_repo_path = "{0}/{1}/{2}".format(conf["pkgbot"]["public-root"], wanted_repo, distro)
             # createrepo
-            createrepo_cmd = ["createrepo", rpm_repo_path]
+            createrepo_cmd = ["createrepo_c", rpm_repo_path]
```

Comment:
This is very Debian-specific and should be fixed downstream (unless this repo specifically only targets Debian&mdash;then it's probably fine).

Reply:
(for reasons discussed off-channel, not considering this an issue)